src/runtime_src/core/include/experimental/xrt_graph.h

```diff
@@ -26,6 +26,205 @@
 
 typedef void *xrtGraphHandle;
```

Comment:
```suggestion
using xrtGraphHandle = void*;
```
but actually you should have something more type-safe like:
```c++
struct xrtGraphHandle {
  void* handle;

  // Some accessors or converters...
};
```

Reply:
This is a C header as well as a C++ header, leave as typedef.

Reply:
OK, then forgot about it (and type-safety...).